layouts/shortcodes/api-reference.html

```diff
@@ -4,4 +4,4 @@
 {{ $textArg := .Get "text" }}
 {{ $page := site.GetPage "page" (printf "%s/%s" $base $pageArg) }}
 {{ $metadata := $page.Params.api_metadata }}
-<a href="{{ $page.URL }}{{if $anchorArg}}#{{ $anchorArg }}{{end}}">{{if $textArg}}{{ $textArg }}{{else if $anchorArg}}{{ $anchorArg }}{{else}}{{ $metadata.kind }}{{end}}</a>
\ No newline at end of file
+<a href="{{ $page.RelPermalink }}{{if $anchorArg}}#{{ $anchorArg }}{{end}}">{{if $textArg}}{{ $textArg }}{{else if $anchorArg}}{{ $anchorArg }}{{else}}{{ $metadata.kind }}{{end}}</a>
```

Comment:
I'd separate this into its own commit, or even into its own PR (the change to this line is a good fix and easy to accept).